mk/config.mk

```diff
@@ -1063,3 +1063,13 @@ CFG_HMAC_64_1024_RANGE ?= n
 # By default use standard pbkdf2 implementation
 CFG_CRYPTO_HW_PBKDF2 ?= n
 $(eval $(call cfg-depends-all,CFG_CRYPTO_HW_PBKDF2,CFG_CRYPTO_PBKDF2))
+
+# Disable the widevine PTA by default
```

Comment:
this should rather describe what `CFG_WIDEVINE_PTA` relates to, not what is its default state.

Same comment for `CFG_WIDEVINE_HUK`. IIUC, `CFG_WIDEVINE_HUK` when enabled, uses the widevine HUK provided by secure DTB as OP-TEE HUK. Maybe using `CFG_WIDEVINE_HUK` puts some constraints how the HUK is derived (`CFG_CORE_HUK_SUBKEY_COMPAT` config switch value).

Reply:
Fixed the comments.
BTW, IIUC, we don't have constraints between `CFG_WIDEVINE_HUK` & `CFG_CORE_HUK_SUBKEY_COMPAT`.

Reply:
OK. I don't really understand why using a Widevine TA mandates OP-TEE to get its HUK from the Widevine integration. And when so, that there is not strict scheme for how this HUK is to be used. But I guess there are considerations I don't catch.